solax/inverters/x1_hybrid_gen5.py

```diff
@@ -0,0 +1,65 @@
+import voluptuous as vol
+
+from solax import utils
+from solax.inverter import Inverter, InverterHttpClient, Method, ResponseParser
+from solax.units import Total, Units
+from solax.utils import div10, div100, pack_u16, to_signed
+
+
+class X1HybridGen5(Inverter):
+    # pylint: disable=duplicate-code
+    _schema = vol.Schema(
+        {
+            vol.Required("type"): vol.All(int, 15),
+            vol.Required(
+                "sn",
+            ): str,
+            vol.Required("ver"): str,
+            vol.Required("Data"): vol.Schema(
+                vol.All(
+                    [vol.Coerce(float)],
+                    vol.Length(min=300, max=300),
```

Comment:
Actually I get this on my Hybrid X1 Gen 4 with updated firmware (at least I suspect so, I just contacted support to update firmware and while they didn't reply anything to me, I suspect they have already done it)